app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<% title t('titles.passwords.forgot') %>
+<% request_id = params[:request_id] || sp_session[:request_id] %>
+
+<%= render 'shared/sp_alert' %>
+
+<h1 class="h3 my0">
+  <%= t('headings.passwords.forgot') %>
+</h1>
+
+<p class="mt-tiny mb0" id="email-description">
+  <%= t('instructions.password.forgot') %>
+</p>
+
+<%= simple_form_for(@password_reset_email_form,
```

Comment:
All the `simple_form_for` calls are about to be renamed in https://github.com/18F/identity-idp/pull/4221 so these are gonna conflict 😬  --- guess you better try to merge first!

Reply:
> All the `simple_form_for` calls are about to be renamed in #4221 so these are gonna conflict 😬 --- guess you better try to merge first!

The race is on! 🏎